ci/deploy.yml

```diff
@@ -182,33 +182,16 @@ release core unix debug deploy:
   before_script: []  # no poetry
   needs:
     - core unix frozen debug build
+    - core unix frozen debug build arm
   script:
     - export VERSION=$(./tools/version.sh core/embed/firmware/version.h)
     - DEST="$DEPLOY_PATH/trezor-emu-core-v$VERSION"
-    - echo "Deploying to $DEST"
+    - DEST_ARM="$DEPLOY_PATH/trezor-emu-core-arm-v$VERSION"
+    - echo "Deploying to $DEST and "$DEST_ARM"
```

Comment:
echo "Deploying to $DEST and $DEST_ARM"

(extra ")

Reply:
Feel free to force-push then!

Reply:
fixed in 6f7b404d0